client/utils/index.js

```diff
@@ -33,6 +33,18 @@ export const getPaymentSettingsUrl = () => {
 	} );
 };
 
+/**
+ * Returns the URL to the list of payment methods page.
+ *
+ * @returns {string} URL to the list of payment methods page.
+ */
+export const getPaymentMethodsUrl = () => {
+	return addQueryArgs( 'admin.php', {
+		page: 'wc-settings',
+		tab: 'checkout',
+	} );
```

Comment:
Just a thought: Should we maybe generate this URL somewhere in PHP? This is similar to https://github.com/Automattic/woocommerce-payments/blob/master/includes/class-wc-payment-gateway-wcpay.php#L37 and even though it's serving a different purpose, it feels like duplication.

Reply:
>  it feels like duplication.

It feels a little bit, but I don't think it's too much of duplication, do you?

I was thinking about alternatives, and what would be the way of passing that url to the client. I have two options in my mind:

1. Set it to the global variable `wcpaySettings`. Not reliable IMO as it's easy to replace with any other script.
2. Return redirect url in response body. Not sure if it's better approach because the redirect url would be returned only when endpoint is called with `accept: false`.

What do you think about options above? Do you have other ideas?

Reply:
My original idea was to localize the ToS script through the `wcpay_tos_settings` global variable (specific for the script), but I've come to agree with your original approach, so let's keep it that way.